docs/installing/index.md

```diff
@@ -49,6 +49,17 @@ Voila! You're now an official WP-CLI user.
 
 Wondering what to do next? Check out the [quick start guide](/docs/quick-start/) for a short introduction and some example usage.
 
+## Updating WP-CLI
+Once WP-CLI is installed you can update it at any time by running `wp cli update`.  
```

Comment:
Can you add another line between the section head and this sentence?

Also, this sentence should communicate that `wp cli update` works for the Phar installation mechanism, but doesn't work for Git or Composer-based installations.